src/testRunner/unittests/tsbuild/containerOnlyReferenced.ts

```diff
@@ -17,10 +17,10 @@ namespace ts {
             ];
         }
 
-        it("verify that subsequent builds after initial build doesnt build anything", () => {
+        it("verify that subsequent builds after initial build doesnt build anything", async () => {

```

Comment:
The tests aren't passing.  I'm not going to fix them until I know there aren't deeper problems with the change.